client/src/components/timeline/revision.js

```diff
@@ -70,8 +96,8 @@ const Revision = ( { side, revision, date, duration, wiki } ) => {
 								<div className="record row justify-content-between">
 									<div className="col-xxl-1 col-xl-2 col-4 align-self-center timestamp">{timestamp.format( 'h:mma' )}</div>
 									<a href={url} className="col-xxl-11 col-xl-10 col-8 d-block content rounded pt-1 pb-1">
-										<span className="d-block title">{revision.title}</span>
-										<span className="d-block comment"><em>{revisionComment}</em></span>
+										<span className="d-block title">{getDisplayTitle( revision.title, revision.comment )}</span>
```

Comment:
`getDisplayTitle()` and `getDisplayComment` could (read: will) be run many times since it is run on render and re-rending is a frequent operation. It _might_ be better to memorize the value in a selector... then again regex is pretty fast, so probably not an issue worth worrying about. :)